web.rb

```diff
@@ -95,7 +99,7 @@ def validateApiKey
 
   content_type :json
   status 200
-  token.to_json
+  return {:secret => token.secret}.to_json
```

Comment:
nit: maybe add a similar comment here

```suggestion
  # Note that we intentionally return only the `secret` from the ConnectionToken object.
  # Returning the entire ConnectionToken object creates a dependency between your
  # backend's Stripe.api_version and your clients, making future upgrades more complicated.
  # https://stripe.com/docs/upgrades#what-changes-does-stripe-consider-to-be-backwards-compatible
  return {:secret => token.secret}.to_json
```

Reply:
Would you also a note to `/create_payment_intent` and `/capture_payment_intent`?

https://github.com/stripe/example-terminal-backend/blob/61d2108a1e5da5d6899d8792b4501104898b77e3/web.rb#L125
https://github.com/stripe/example-terminal-backend/blob/61d2108a1e5da5d6899d8792b4501104898b77e3/web.rb#L142

I figured 3 examples that have an explicit return format and 2 examples with warnings would be sufficient

Reply:
ah, forgot about those cases too. up to you in the end! i think it's also fine as-is.